packages/katello/katello/katello.spec

```diff
@@ -180,6 +178,9 @@ Provides a federation of katello services
 # the files section is empty, but without it no RPM will be generated
 
 %changelog
+* Tue Dec 15 2020 Evgeni Golov - 3.18.0-1
+- Release katello 3.18.0
+
 * Thu Nov 19 2020 Evgeni Golov - 3.18.0-0.1.rc2
```

Comment:
This is a confusing changelog with 3 times `Release Katello 3.18.0`

Reply:
*whistle* we'd need to fix obal to add the prerelease to the message